### DISCUSSION TITLE
Voiceovers are broken in Python 3.11

### DISCUSSION THREAD

**Discussion by Joey451-OG** [ManimCommunity/manim#3202] [General]
Open AI's [Whisper](https://github.com/openai/whisper) does not work for python 3.11 yet. They are currently working on [it](https://github.com/openai/whisper/pull/1171) (well actually they are waiting on another repo to finish [here](https://github.com/numba/numba/issues/8841)). Since the `RecoderScene` from `manim_voiceover.services.recorder` relies on the Open AI project, there really isn't anything this repo can do right now. I just made this post to notify people.

**Comment by anni-anni** [comment]
The OpenAI Whisper API falters from time to time as it's considerably new and needs a ton of bug fixes...meanwhile I try using different [AI text to speech](https://wavel.ai/solutions/realistic-text-to-speech-online/) generators online that are compatible with my iOS and the current API I'm using has over 250+ voices and accents for TTS! That's really helpful!

**Comment by 100Khushi** [comment]
Hi,

It's great that you're exploring alternative [AI text to speech](https://wavel.ai/solutions/realistic-text-to-speech-online) generators for your iOS. Having over 250+ voices and accents is indeed impressive, offering a wide range of options. If you ever need more diversity or specific characteristics in your TTS, you might want to experiment with other generators too. 
Happy experimenting!

**Comment by Joey451-OG** [comment]
Closing this as it seems like bots started advertising here.
